examples/fatigue/static_arm_with_fatigue.py

```diff
@@ -0,0 +1,161 @@
+"""
+This is a basic example on how to use biorbd model driven by muscle to perform an optimal reaching task.
+Fatigue is applied on muscles. The arms must reach a marker placed upward in front while minimizing
+the muscles activity and fatigue.
+Please note that using show_meshes=True in the animator may be long due to the creation of a huge CasADi graph of the
+mesh points.
+"""
+
+import biorbd_casadi as biorbd
+
+from bioptim.misc.enums import Fatigue
+from bioptim import (
+    OptimalControlProgram,
+    ObjectiveList,
+    ObjectiveFcn,
+    Dynamics,
+    DynamicsFcn,
+    QAndQDotBounds,
+    InitialGuess,
+    OdeSolver,
+    Bounds,
+    Constraint,
+    ConstraintFcn,
+    XiaFatigueDynamicsList,
+    XiaFatigueStateInitialGuess,
+    XiaFatigueStateBounds,
+    XiaFatigueControlsBounds,
+    XiaFatigueControlsInitialGuess,
+    Node,
+    Axis,
+)
+
+
+def prepare_ocp(
+    biorbd_model_path: str,
+    final_time: float,
+    n_shooting: int,
+    ode_solver: OdeSolver = OdeSolver.RK4(),
+    with_residual_torque: bool = False,
+    fatigue: list = None,
+) -> OptimalControlProgram:
+    """
+    Prepare the ocp
+    Parameters
+    ----------
+    biorbd_model_path: str
+        The path to the bioMod
+    final_time: float
+        The time at the final node
+    n_shooting: int
+        The number of shooting points
+    ode_solver: OdeSolver
+        The ode solver to use
+    with_residual_torque: bool
+        True if we use residual torque
+    fatigue: list
+        The type of fatigue applied on the system
+    Returns
+    -------
+    The OptimalControlProgram ready to be solved
+    """
+
+    biorbd_model = biorbd.Model(biorbd_model_path)
+
+    n_muscles = biorbd_model.nbMuscleTotal()
+    muscle_min, muscle_max, muscle_init = 0, 1, 0.3
+
+    tau_min, tau_max, tau_init = -1, 1, 0
+
+    # Dynamics
+    dynamics = Dynamics(
+        DynamicsFcn.MUSCLE_DRIVEN, expand=False, fatigue=fatigue, with_residual_torque=with_residual_torque
+    )
+
+    # Add objective functions
+    objective_functions = ObjectiveList()
+    objective_functions.add(ObjectiveFcn.Lagrange.MINIMIZE_CONTROL, key="muscles", weight=1)
+    objective_functions.add(
+        ObjectiveFcn.Mayer.SUPERIMPOSE_MARKERS, first_marker="target", second_marker="COM_hand", weight=0.01
+    )
+    objective_functions.add(ObjectiveFcn.Lagrange.MINIMIZE_STATE, key="muscles_mf", weight=10000)  # Minimize fatigue
```

Comment:
Could you add a TODO in MINIMIZE_STATE so the user does not have to know `_mf`. Or creating a MINIMIZE_FATIGUE that calls MINIMIZE_STATE, but appends automatically the `_mf`

Reply:
I have done it already